renderdoc/driver/gl/gl_pixelhistory.cpp

```diff
@@ -151,18 +175,18 @@ void QueryPostModPixelValues(WrappedOpenGL *driver, GLPixelHistoryResources &res
     driver->ReplayLog(modEvents[i].eventId, modEvents[i].eventId, eReplay_Full);
 
     // read the post mod pixel value into the history event
-    PixelModification mod;
     ModificationValue modValue;
     PixelValue pixelValue;
     driver->glReadPixels(x, y, 1, 1, eGL_RGBA, eGL_FLOAT, (void *)pixelValue.floatValue.data());
     driver->glReadPixels(x, y, 1, 1, eGL_DEPTH_COMPONENT, eGL_FLOAT, (void *)&modValue.depth);
     driver->glReadPixels(x, y, 1, 1, eGL_STENCIL_INDEX, eGL_INT, (void *)&modValue.stencil);
     modValue.col = pixelValue;
 
-    RDCEraseEl(mod);
-    mod.eventId = modEvents[i].eventId;
-    mod.postMod = modValue;
-    history.push_back(mod);
+    for(; historyIndex < history.size() && modEvents[i].eventId == history[historyIndex].eventId;
+        ++historyIndex)
+    {
+      history[historyIndex].postMod = modValue;
+    }
```

Comment:
We were originally going to query per fragment values in this PR but then decided to split it, so this per fragment history got left over. We can take this out for this PR to avoid confusion, and just treat each event as a blob until we have the fragment queries ready.